doc/neo-tree.txt

```diff
@@ -255,6 +255,9 @@ For example:
 The above config will map `A` to command_a for all sources except for
 filesystem, which will use command_b instead.
 
+If you want to disable all default mappings, you can set
+`disable_default_mappings = false` in your config.
```

Comment:
I think this is backwards.

Reply:
Fixed, and renamed to `use_default_mappings`.